src/main/scala/strawman/collection/Seq.scala

```diff
@@ -72,6 +72,14 @@ trait IndexedSeqOps[+A, +CC[X] <: IndexedSeq[X], +C] extends Any with SeqOps[A,
     def length: Int = self.length
     def apply(i: Int): A = self(i)
   }
+
+  /** A collection containing the last `n` elements of this collection. */
+  override def takeRight(n: Int): C = fromSpecificIterable(view.takeRight(n))
```

Comment:
Ah thanks, I missed that difference between both implementations. Indeed imho that would make sense to use `view.xxx` everywhere in place of `View.Xxx`. That might have an impact on performance, since that’s one more indirection level but I wonder if the JVM can always inline that.